api/src/main/java/com/lifegraph/team20/payload/request/SearchGraphInfo.java

```diff
@@ -1,50 +1,50 @@
-package com.lifegraph.team20.payload.request;
-
-import java.sql.Timestamp;
-
-public class SearchGraphInfo {
-	private String name;
-	private int user_id;
-	private Timestamp created_at;
-	private Timestamp updated_at;
-
-	public SearchGraphInfo( String name, int user_id, Timestamp created_at, Timestamp updated_at) {
-		this.setName(name);
-		this.setId(user_id);
-		this.setCreated(created_at);
-		this.setUpdated(updated_at);
-	}
-
-	public String getName() {
-		return name;
-	}
-
-	public void setName(String name) {
-		this.name = name;
-	}
-
-	public int getId() {
-		return user_id;
-	}
-
-	public void setId(int user_id) {
-		this.user_id = user_id;
-	}
-
-	public Timestamp getCreated() {
-		return created_at;
-	}
-
-	public void setCreated(Timestamp created_at) {
-		this.created_at = created_at;
-	}
-
-	public Timestamp getUpdated() {
-		return updated_at ;
-
-	}
-
-	public void setUpdated(Timestamp updated_at) {
-		this.updated_at = updated_at;
-	}
-}
\ No newline at end of file
+//package com.lifegraph.team20.payload.request;
+//
+//import java.sql.Timestamp;
+//
+//public class SearchGraphInfo {
+//	private String name;
+//	private int user_id;
+//	private Timestamp created_at;
+//	private Timestamp updated_at;
+//
```

Comment:
// @Data
//	public SearchGraphInfo( String name, int user_id, Timestamp created_at, Timestamp updated_at) {
//		this.setName(name);
//		this.setId(user_id);
//		this.setCreated(created_at);
//		this.setUpdated(updated_at);
//	}